src/video_emotion/facial_recognition/facial_recognition.py

```diff
@@ -57,7 +75,19 @@ def analyze_video(video_path, time_range=None):
     return dict_of_faces
 
 
-def analyze_frame(frame):
+def analyze_frame(frame) -> [[int, int]]:
+    """Analyses a single frame finding faces returning as a coordinate
```

Comment:
Should probably be separated by commas. "Analyses a single frame, finding faces and returning them as a coordinate"